config/exchanges/LS_ZA.yaml

```
@@ -0,0 +1,4 @@
+lonlat:
+  - 27.454639
+  - -29.297917
+rotation: 87
```

```suggestion
rotation: -87
```
I think we just need to rotate this. Otherwise it looks good to me.

---

The exchange is intended to go from South Africa (source) into Lesotho (destination) (on the map, that will appear as going left-to-right with a slight angle, hence the 87deg instead of 90deg). 

Based on the other exchange configurations (such as South Australia -> Victoria, which goes left-to-right), a positive rotation value would direct it this way. Similarly, a negative value (such as Great Britain -> Ireland) would go right-to-left. 

Have I misunderstood the intended exchange direction and/or how the rotation values work?

Thanks again for your help, and please let me know if you need any more info from me to help clarify this!

---

No you have not, but the exchange direction is defined alphabetically so currently the exchange is going from LS into ZA.

But the rotation currently shows the opposite.

---

Ahhh ok, thanks for clarifying! 

Should I rename the file to ZA_LS.ymal, or should I keep this file (and correct the rotation) and add a new file for ZA_LS.yaml?

---

As far as I know there are no exchanges from Lesotho into South Africa, so I don’t think a LS-ZA exchange config is necessary.

---

Thanks for the clarification Viktor; that makes sense.

I’ve updated the rotation and aligned this branch with the main branch, so the pull request should be ready for review again.

Thanks once again for your help, I really appreciate it!